<launch>
  
  <include file="$(find state_manager)/launch/stateManager.launch" />
  
  <include file="$(find openni_launch)/launch/openni.launch" >
    <arg name="camera" value="kinect" />
    <arg name="publish_tf" value="false" />
    <arg name="depth_registration" value="true" />
  </include>
  <param name="kinect/topic_name" value="/kinect/rgb/image_raw" />
  
  <include file="$(find pandora_vision_landoltc)/launch/landoltc_node.launch" />
  
  <include file="$(find pandora_robots_description)/launch/gears/gears_description.launch" />
  
</launch>  
